airflow/providers/google/marketing_platform/hooks/display_video.py

```diff
@@ -228,7 +228,7 @@ def get_sdf_download_operation(self, operation_name: str):
         result = (
             self.get_conn_to_display_video()  # pylint: disable=no-member
             .sdfdownloadtasks()
-            .operation()
+            .operations()
```

Comment:
Does this mean that the operator was not working previously? 👀

Reply:
Sorry for the late response! I just saw this. Yes, the operator was previously not working at all. I identified several different issues all causing the operator to not work and submitted in a few additional patches.

Reply:
This change already covered in https://github.com/apache/airflow/pull/13703